unique_sdk/README.md

```diff
@@ -690,8 +690,8 @@ Allows for uploading files that then get ingested in a scope or a chat.
 
 ```python
 createdContent = upload_file(
-    userId,
-    companyId,
+    companyId=companyId,
```

Comment:
Is the order of parameters here critical? If yes, is this order correct now. Looks like it was flipped.

Reply:
Now the order is correct.

Reply:
Yes order is crucial.